solc/CommandLineInterface.cpp

```diff
@@ -420,7 +420,16 @@ void CommandLineInterface::readInputFilesAndConfigureRemappings()
 bool CommandLineInterface::parseLibraryOption(string const& _input)
 {
 	namespace fs = boost::filesystem;
-	string data = fs::is_regular_file(_input) ? contentsString(_input) : _input;
+	string data = _input;
+	try
+	{
+		if (fs::is_regular_file(_input))
```

Comment:
Does this mean, by `touch`ing the file named `_input`, you can change the behavior of the compiler?  Is one able to invoke the Solidity compiler with the same arguments repeatedly, to perform stateful computation?